addons/web/static/src/views/form/form_controller.scss

```diff
@@ -728,6 +728,12 @@
         }
     }
 
+    .o_field_percentage.oe_inline {
+        &, input {
+            max-width: 7ch;
```

Comment:
Isn't another unit like rem preferred? I don't know much about css but I haven't seen ch very often, so Xavier will decide.

Reply:
I don't think it is a good idea to fix the width in generic way. IMO, it really depend how we would like to display the field. 🤔 

Why not doing the fix in the our view in XML? 🧐

Reply:
I think `oe_inline` on field should reduce it width as this is common issue for all percentage field like it is done for all the other field(integer, float, date, etc...) above this :thinking:

Reply:
Okay, I agree good idea. 🙂